test/Microsoft.AspNet.Mvc.FunctionalTests/project.json

```diff
@@ -43,7 +43,8 @@
         "MvcTagHelpersWebSite": "1.0.0",
         "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-*",
         "xunit.runner.kre": "1.0.0-*",
-        "Microsoft.AspNet.WebUtilities": "1.0.0-*"
+        "Microsoft.AspNet.WebUtilities": "1.0.0-*",
+"FormatFilterWebSite": "1.0.0-*"
```

Comment:
Put this in order with the others please, this list is mostly alphabetized already